layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "shaderStorageBufferArrayNonUniformIndexing": 0,
+        "shaderStorageImageArrayNonUniformIndexing": 0,
+        "shaderInputAttachmentArrayNonUniformIndexing": 0,
+        "shaderUniformTexelBufferArrayNonUniformIndexing": 0,
+        "shaderStorageTexelBufferArrayNonUniformIndexing": 0,
+        "descriptorBindingUniformBufferUpdateAfterBind": 0,
+        "descriptorBindingSampledImageUpdateAfterBind": 0,
+        "descriptorBindingStorageImageUpdateAfterBind": 0,
+        "descriptorBindingStorageBufferUpdateAfterBind": 0,
+        "descriptorBindingUniformTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingStorageTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingUpdateUnusedWhilePending": 0,
+        "descriptorBindingPartiallyBound": 0,
+        "descriptorBindingVariableDescriptorCount": 0,
+        "runtimeDescriptorArray": 0
+    },
+    "VkPhysicalDeviceHostQueryResetFeatures": {
+        "hostQueryReset": 0
+    },
+    "VkPhysicalDeviceImagelessFramebufferFeatures": {
+        "imagelessFramebuffer": 0
+    },
+    "VkPhysicalDeviceMultiviewFeatures": {
+        "multiview": 1,
+        "multiviewGeometryShader": 0,
+        "multiviewTessellationShader": 0
+    },
+    "VkPhysicalDeviceProtectedMemoryFeatures": {
+        "protectedMemory": 0
+    },
+    "VkPhysicalDeviceSamplerYcbcrConversionFeatures": {
+        "samplerYcbcrConversion": 0
+    },
+    "VkPhysicalDeviceScalarBlockLayoutFeatures": {
+        "scalarBlockLayout": 0
+    },
+    "VkPhysicalDeviceSeparateDepthStencilLayoutsFeatures": {
+        "separateDepthStencilLayouts": 0
+    },
+    "VkPhysicalDeviceShaderAtomicInt64Features": {
+        "shaderBufferInt64Atomics": 1,
+        "shaderSharedInt64Atomics": 1
+    },
+    "VkPhysicalDeviceShaderDrawParametersFeatures": {
+        "shaderDrawParameters": 0
+    },
+    "VkPhysicalDeviceShaderFloat16Int8Features": {
+        "shaderFloat16": 0,
+        "shaderInt8": 0
+    },
+    "VkPhysicalDeviceShaderSubgroupExtendedTypesFeatures": {
+        "shaderSubgroupExtendedTypes": 0
+    },
+    "VkPhysicalDeviceTimelineSemaphoreFeatures": {
+        "timelineSemaphore": 0
+    },
+    "VkPhysicalDeviceUniformBufferStandardLayoutFeatures": {
+        "uniformBufferStandardLayout": 1
+    },
+    "VkPhysicalDeviceVariablePointersFeatures": {
+        "variablePointersStorageBuffer": 0,
+        "variablePointers": 0
+    },
+    "VkPhysicalDeviceVulkanMemoryModelFeatures": {
+        "vulkanMemoryModel": 1,
+        "vulkanMemoryModelDeviceScope": 0,
+        "vulkanMemoryModelAvailabilityVisibilityChains": 0
+    },
+    "Vulkan12Features": {
+        "samplerMirrorClampToEdge": 0,
+        "shaderOutputViewportIndex": 0,
+        "shaderOutputLayer": 0,
+        "subgroupBroadcastDynamicId": 1,
+        "drawIndirectCount": 0,
+        "descriptorIndexing": 0,
+        "samplerFilterMinmax": 0
+    },
+    "Vulkan12Properties": {
+        "framebufferIntegerColorSampleCounts": 1
```

Comment:
Effectively, this is an array but I agree it's 1. 
So it should be "framebufferIntegerColorSampleCounts": [1]

Reply:
This is not defined as an array according to the schema.
According to the scheme, this should be represented as type `VkSampleCountFlags`, which follows a definition chain down to an `enum` defined as type `int32_t` or a JSON `number` within the range [-2147483648, 2147483647].

This is to follow the philosophy that types in the config file should be stored as similar as possible to their types when used in the application to make programmatically reading and writing them relatively simple.

Again, this is a comment on possible improvements to config file readability, which this PR is not meant to address.